### PR TITLE
Add VNNI support warning to resnet50_benchmark

### PR DESCRIPTION
**Pull request by mgoin**

### CODE REVIEW

File: examples/benchmark/resnet50_benchmark.py [hunk 1]
```diff
@@ -146,6 +146,12 @@ def main():
     )
     print(f"ResNet-50 v1 Pruned Moderate FP32 {results}")
 
+    if not VNNI:
+        print(
+            "WARNING: VNNI instructions not detected, "
+            "quantization speedup not well supported"
```
Comment (bfineran) [inline examples/benchmark/resnet50_benchmark.py:152]:
```suggestion
            "quantization (INT8) speedup not well supported"
```

**Review by bfineran — DISMISSED** [review]
added a suggestion to specify INT8 quantization since the script logs specify INT8 ResNet-50 instead of "quantized". am fine landing as-is though - up to you

**Review by bfineran — APPROVED** [review]

**Review by rahul-tuli — APPROVED** [review]
LGTM!

**Comment by markurtz** [comment]
Is there anyway we can make this detection more generic and move it inside of DeepSparse by detecting quantized portions of the graph? People are going to run into this in pretty much every example we have along with their own models

**Comment by mgoin** [comment]
> Is there anyway we can make this detection more generic and move it inside of DeepSparse by detecting quantized portions of the graph? People are going to run into this in pretty much every example we have along with their own models

yes we'll work on getting this into the engine as a one-time warning, like the splash message

**Pull request merged**